taskcluster/mozillavpn_taskgraph/transforms/beetmover.py

```diff
@@ -0,0 +1,99 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import os.path
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_beetmover_worker_config(config, tasks):
+    for task in tasks:
+        app_name = "vpn"
+        worker_type = task["worker-type"]
+        task_name = task["name"]
+        task_label = f"{worker_type}-{task_name}"
+        l3_relpro = (
```

Comment:
Is this supposed to be `is_relpro` ? The variable name is confusing to me.

Reply:
Is L3 relpro, but I see what u mean, will probably land a rename with the windows task